sandbox/version-control-manager/src/main/java/com/vikas/method_parser/version_control_manager/details/fetcher/cmd/CommandLineInterface.java

```diff
@@ -2,7 +2,16 @@
 
 import com.vikas.method_parser.version_control_manager.details.fetcher.model.UserInput;
 
+/**
+ * To get user input.
```

Comment:
This class will do much more than user input. It will handle all the command-line interactions.